app/layout.tsx

```diff
@@ -1,32 +1,35 @@
-import type { Metadata } from 'next'
-import { Inter } from 'next/font/google'
-import './globals.css'
-import Navbar from '../components/Navbar'
-import Footer from '../components/Footer'
-import AuthProvider from '../components/AuthProvider'
+'use client';
```

Comment:
Is it a liitle bit weird adding the use client in the root layout? That means we don't need next js at all, because all components are client side render?

Reply:
Got it, I've switched it to the SSR component.

Initially, I used 'use client' because I misunderstood a sentence from the documentation. It stated, "Before Next.js 13, we typically added the Provider in the _app.tsx file. However, in the latest version of Next, all pages are server-side rendered by default. To make Redux work, we need client-side components."

However, I noticed that I've created a CSR provider to address this issue. Therefore, it is no longer necessary to use ‘use client’ in layout components.